src/ray/common/task/scheduling_resources.cc

```diff
@@ -189,6 +189,11 @@ const ResourceSet ResourceSet::GetNumCpus() const {
   return cpu_resource_set;
 }
 
+double ResourceSet::GetNumCpusDouble() const {
```

Comment:
no strong opinion here, but just feel GetNumCpusDouble looks bad.

One way to work around for this one is: 

```
template<typename T>
T ResourceSet::GetNumCpus() const {
}
```